### PR TITLE
CoreConcept: PSTs #66

### PR DESCRIPTION
**Pull request by felwintrr**

### CODE REVIEW

File: docs/src/concepts/psts.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Overview
+
+---
+
+Profit Sharing Tokens (PSTs) are tokens issued by a community or protocol created on the permaweb. The protocol issuing a PST is often referred to as a ‘Profit Sharing Community’ (PSC).
+
```
Comment (twilson63) [inline docs/src/concepts/psts.md:6]: PSTs do not have to be issued by a PSC, they can be created as SWT or SmartWeaveTokens:

* balances object
* name string
* ticker string

Comment (felwintrr) [reply docs/src/concepts/psts.md:6]: Will adjust, thanks

**Review by twilson63 — COMMENTED** [review]
Added a comment, but I think others should chime in as well.

**Review by twilson63 — APPROVED** [review]
LGTM

**Pull request merged**